lib/cuffsert/yaml-ext.rb

```diff
@@ -0,0 +1,22 @@
+require 'yaml'
+
+module YAML
+  %w[Ref].each do |name|
+    add_domain_type('cuffsert', name) do |tag, value|
+      {name => value}
+    end
+  end
+
+  add_domain_type('cuffsert', 'GetAtt') do |_, value|
+    {'Fn::GetAtt' => value.to_s.split('.')}
```

Comment:
`!GetAtt [Distribution, DomainName]` seems to be supported too, although not documented. I can live with not supporting it, but I actually have a template that's using that syntax.

Reply:
I think it's shorthand for

```yaml
DNSName: !GetAtt
  - Distribution
  - DomainName
```

Reply:
If you say so :) . Supported now.